_posts/2015-02-09-geoblacklight-overview.markdown

```diff
@@ -32,17 +32,19 @@ That's ok. GIS data indexed into GeoBlacklight becomes progressively more useful
 
 [The GeoBlacklight Schema, Version 1.0](https://github.com/geoblacklight/geoblacklight/blob/master/schema/geoblacklight-schema.md) uses a field `dct_references_s` to define external services and references. See the [references.md](https://github.com/geoblacklight/geoblacklight/blob/master/schema/references.md) portion of the repository for a list of possible key:value pairs and instructions on how to apply them.
```

Comment:
Links are to the Markdown files, which are possibly a bit out of date. We could update this later though.

Reply:
I've linked the wiki pages now. Thanks for the suggestion.